.github/workflows/benchmark.yml

```diff
@@ -7,6 +7,10 @@ on:
     paths-ignore:
       - 'website/**'
 
+concurrency:
```

Comment:
all of these are not really part of the PR?

Reply:
I wrote about them in the PR description. It helps us to cancel previous workflow runs even if they are queued, because the current logic needs other run to cancel it.